### DISCUSSION TITLE
bpo-22789: Compress the marshalled data in PYC files

### DISCUSSION THREAD

**Discussion by iritkatriel** [faster-cpython/ideas#122] [Ideas]
Raymond create this issue: https://bugs.python.org/issue22789

**Comment by warsaw** [comment]
Could be interesting to explore further.  I'm much more interested in speed improvements, especially at start up time than disk space per se.

Some [updated](http://mattmahoney.net/dc/text.html) [benchmarks](https://linuxreviews.org/Comparison_of_Compression_Algorithms) to investigate.
